Dockerfile

```
@@ -17,14 +19,18 @@ COPY apis/ apis/
 COPY controllers/ controllers/
 COPY pkg/ pkg/
 
+# Add the local bundle
+ADD https://github.com/opendatahub-io/odh-manifests/tarball/master $LOCAL_BUNDLE
```

```suggestion
ADD https://github.com/opendatahub-io/odh-manifests/tarball/master $LOCAL_BUNDLE
RUN chmod g+r $LOCAL_BUNDLE
```

We need to make this file writable by the `root` group so that it can be read as a local manifest